cmd/backup/main.go

```diff
@@ -4,21 +4,33 @@
 package main
 
 import (
+	"flag"
 	"fmt"
+	"log/slog"
 	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/robfig/cron/v3"
 )
 
-func main() {
-	s, err := newScript()
+func runBackup(c *Config) (ret error) {
```

Comment:
```suggestion
func runScript(c *Config) (ret error) {
```

How about naming this `runScript` as it constructs a `script` object and then runs it?

Reply:
Done